app/src/views/OperationalLearnings/i18n.json

```
@@ -0,0 +1,7 @@
+{
+    "namespace": "operational-learnings",
```

```suggestion
    "namespace": "operationalLearnings",
```
we use camel case for variable names in IFRC Go